install-solr.sh

```diff
@@ -4,13 +4,13 @@ if [ $# -gt 0 ]; then
 fi
 
 wget https://archive.apache.org/dist/tika/tika-app-1.20.jar -O bin/tika-app-1.20.jar
-wget https://archive.apache.org/dist/lucene/solr/6.6.5/solr-6.6.5.tgz
-tar xfz solr-6.6.5.tgz
-mv solr-6.6.5 solr
+wget https://archive.apache.org/dist/lucene/solr/7.7.3/solr-7.7.3.tgz -O solr-7.7.3.tgz
+tar xfz solr-7.7.3.tgz
+mv solr-7.7.3 solr
 cd solr
 mkdir -p server/ez/template
 cp -R ../migrations/solr/* server/ez/template
-cp server/solr/configsets/basic_configs/conf/{currency.xml,solrconfig.xml,stopwords.txt,synonyms.txt,elevate.xml} server/ez/template
+cp example/example-DIH/solr/solr/conf/{currency.xml,solrconfig.xml,stopwords.txt,synonyms.txt,elevate.xml} server/ez/template
```

Comment:
Some config files from templates should be sync. with ezplatform-solr-search-engine e.g. https://github.com/ezsystems/ezcommerce/blob/master/migrations/solr/schema.xml. More infomation in PR: ezsystems/ezplatform-solr-search-engine#136

Reply:
I'm not sure if we can safely use config templates from ezplatform-solr-search-engine as SiSo code is not relying on it and they use their own SOLR implementation. I don't know if simple replacement will guarantee their code to work properly. I think it would be safer to keep it as it for the time being.